src/clr-angular/data/datagrid/providers/selection.ts

```diff
@@ -198,6 +222,10 @@ export class Selection<T = any> {
     if (value === this._currentSingle) {
       return;
     }
+    // locked items can't be selected
```

Comment:
This is still here, we need to remove this and the next comment below.

We basically only want to prevent users from selecting/unselecting a locked item. However, an application might want to programmatically change the selected state of a locked row (I can't think of many great scenarios but I still believe we should allow it).

Reply:
Not sure why this is visible, cause on my file it's not. it's something wrong with the diffs here

Reply:
<img width="675" alt="Screen Shot 2019-09-26 at 10 10 58 AM" src="https://user-images.githubusercontent.com/204564/65666265-f4722f80-e045-11e9-84a4-3e06333fd217.png">